modules/psap-node-feature-discovery-topology-updater-command-reference.adoc

```diff
@@ -0,0 +1,190 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_content-type: REFERENCE
+[id="nfd-topology-updater-command-line-flags_{context}"]
+= NFD Topology Updater command line flags
+
+[discrete]
+[id="nfd-topology-updater-help_{context}"]
+== -h, -help
+
+Print usage and exit.
+
+[discrete]
+[id="nfd-topology-updater-version_{context}"]
+== -version
+
+Print version and exit.
+
+[discrete]
+[id="nfd-topology-updater-server_{context}"]
+== -server
+
+The `-server` flag specifies the address of the nfd-master endpoint where to connect to.
```

Comment:
Can "where" be removed in this sentence, or modified somehow? Sounds a little off as it is now.